README


This archive contains the distribution Dist-Zilla-PluginBundle-Author-JQUELIN,
version 3.003:

  Build & release a distribution like jquelin

This software is copyright (c) 2010 by Jerome Quelin.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


This README file was generated by Dist::Zilla::Plugin::Readme v5.013.



Changes
Changelog for Dist-Zilla-PluginBundle-JQUELIN

3.003     2014-02-24 15:25:25+01:00 Europe/Paris
 - no need to try to load plugin
 - doc: refer to metacpan

3.002     2012-10-09 11:23:57 Europe/Paris
 - readme.mkdn should be committed in git

3.001     2012-10-09 10:52:38 Europe/Paris
 - readme.mkdn should be created in root dir

3.000     2012-10-04 15:01:33 Europe/Paris
 - using Git::NextVersion instead of AutoVersion
 - removed the "releases" branch
 - added: Covenant, HelpWanted, ReadmeMarkdownFromPod, TestRelease
 - removed: HasVersionTests, Test::Kwalitee, Test::MinimumVersion

**

2.120971  2012-04-06 12:01:02 Europe/Paris
 - example updated

2.120970  2012-04-06 11:51:47 Europe/Paris
 - renamed to Dist::Zilla::PluginBundle::Author::JQUELIN
 - plugin options now passed as Plugin.option (only one pluginbundle
   variable available: weaver)
 - update renamed test plugins to their new names

**

1.111710  2011-06-20 15:37:36 Europe/Paris
 - new skip_compile parameter

1.110730  2011-03-14 11:13:14 Europe/Paris
 - using Git::CommitBuild for releases

1.110700  2011-03-11 16:03:58 Europe/Paris
 - removed PortabilityTests

1.110010  2011-01-01 14:42:27 Europe/Paris
 - added PruneFiles to remove vim/emacs backup files
 - using ReportVersions::Tiny for smaller footprint / better output
 - using AutoPrereqs instead of AutoPrereq (to be deprecated)
 - removed CriticTests that fail quite too often
 - removed UnusedVarsTests that is not that useful
 - removed MetaTests that is not that useful (dzil generates correct meta)

1.101620  2010-06-11 17:55:23 Europe/Paris
 - updated to match dzil v4
 - nextrelease now forced to europe/paris timezone

1.100972  2010-04-07 16:59:01 Europe/Paris
 - remove testrelease
 - pod update

1.100971  2010-04-07 16:51:15 Europe/Paris
 - added kwalitee tests

1.100970  2010-04-07 16:47:21 Europe/Paris
 - added some test plugins
 - added meta information plugins

1.100900  2010-03-31 10:01:46 Europe/Paris
 - port to dzil v2

1.100660  2010-03-07 10:41:56 Europe/Paris
 - provide shorter names for included plugins
 - provide dist.ini example

1.100220  2010-01-22 16:36:06 Europe/Paris
- new param fake_home for compiletests

1.100130  2010-01-13 15:14:47 Europe/Paris
 - adding prereqs on plugins used
 - prepender plugin now defaults copyright to true

1.100081  2010-01-08 17:59:21 Europe/Paris
 - new param skip_prereq to feed autoprereq plugin

1.100080  2010-01-08 11:12:49 Europe/Paris
 - new param weaver to load taskweaver

1.100060  2010-01-06 13:11:58 Europe/Paris
 - initial release


Build.PL

# This file was automatically generated by Dist::Zilla::Plugin::ModuleBuild v5.013.
use strict;
use warnings;

use Module::Build 0.3601;


my %module_build_args = (
  "build_requires" => {
    "Module::Build" => "0.3601"
  },
  "configure_requires" => {
    "Module::Build" => "0.3601"
  },
  "dist_abstract" => "Build & release a distribution like jquelin",
  "dist_author" => [
    "Jerome Quelin"
  ],
  "dist_name" => "Dist-Zilla-PluginBundle-Author-JQUELIN",
  "dist_version" => "3.003",
  "license" => "perl",
  "module_name" => "Dist::Zilla::PluginBundle::Author::JQUELIN",
  "recommends" => {},
  "recursive_test_files" => 1,
  "requires" => {
    "Class::MOP" => 0,
    "Dist::Zilla::Plugin::AutoPrereqs" => 0,
    "Dist::Zilla::Plugin::Bugtracker" => 0,
    "Dist::Zilla::Plugin::CheckChangeLog" => 0,
    "Dist::Zilla::Plugin::Covenant" => 0,
    "Dist::Zilla::Plugin::ExecDir" => 0,
    "Dist::Zilla::Plugin::ExtraTests" => 0,
    "Dist::Zilla::Plugin::GatherDir" => 0,
    "Dist::Zilla::Plugin::Git::Check" => 0,
    "Dist::Zilla::Plugin::Git::Commit" => 0,
    "Dist::Zilla::Plugin::Git::NextVersion" => 0,
    "Dist::Zilla::Plugin::Git::Push" => 0,
    "Dist::Zilla::Plugin::Git::Tag" => 0,
    "Dist::Zilla::Plugin::HelpWanted" => 0,
    "Dist::Zilla::Plugin::Homepage" => 0,
    "Dist::Zilla::Plugin::License" => 0,
    "Dist::Zilla::Plugin::Manifest" => 0,
    "Dist::Zilla::Plugin::ManifestSkip" => 0,
    "Dist::Zilla::Plugin::MetaConfig" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaProvides::Package" => 0,
    "Dist::Zilla::Plugin::MetaYAML" => 0,
    "Dist::Zilla::Plugin::ModuleBuild" => 0,
    "Dist::Zilla::Plugin::NextRelease" => "2.101230",
    "Dist::Zilla::Plugin::PkgVersion" => 0,
    "Dist::Zilla::Plugin::PodCoverageTests" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::PodWeaver" => 0,
    "Dist::Zilla::Plugin::Prepender" => "1.100130",
    "Dist::Zilla::Plugin::PruneCruft" => 0,
    "Dist::Zilla::Plugin::PruneFiles" => 0,
    "Dist::Zilla::Plugin::Readme" => 0,
    "Dist::Zilla::Plugin::ReadmeMarkdownFromPod" => 0,
    "Dist::Zilla::Plugin::ReportVersions::Tiny" => 0,
    "Dist::Zilla::Plugin::Repository" => 0,
    "Dist::Zilla::Plugin::ShareDir" => 0,
    "Dist::Zilla::Plugin::TaskWeaver" => 0,
    "Dist::Zilla::Plugin::Test::Compile" => "1.100220",
    "Dist::Zilla::Plugin::TestRelease" => 0,
    "Dist::Zilla::Plugin::UploadToCPAN" => 0,
    "Dist::Zilla::Role::PluginBundle" => 0,
    "Dist::Zilla::Role::PluginBundle::Config::Slicer" => 0,
    "Moose" => 0,
    "Moose::Autobox" => 0,
    "perl" => "5.008",
    "strict" => 0,
    "warnings" => 0
  },
  "script_files" => [],
  "test_requires" => {
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.88"
  }
);


my %fallback_build_requires = (
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Module::Build" => "0.3601",
  "Test::More" => "0.88"
);


unless ( eval { Module::Build->VERSION(0.4004) } ) {
  delete $module_build_args{test_requires};
  $module_build_args{build_requires} = \%fallback_build_requires;
}

my $build = Module::Build->new(%module_build_args);

$build->create_build_script;
